app/models/amusement_park.rb

```diff
@@ -1,3 +1,11 @@
 class AmusementPark < ApplicationRecord
   has_many :rides
+  has_many :mechanics, through: :rides
+
+  validates :name, presence: true
```

Comment:
Nice inclusion of validations! We don't expect you to set up validations in an IC but you're welcome to do so for practice/fun!